README.md

```diff
@@ -52,230 +43,171 @@ foreach (var result in searchResults)
 
 ## Supported platforms
 
-.NET 4.5/Mono 3.4, PCL (Windows 8.1, Windows Phone 8.1, iOS (via Xamarin.iOS), Android (via Xamarin.Android)
+This SDK supports .NET 4.5/Mono 3.4, PCL (Windows 8.1, Windows Phone 8.1, iOS (via Xamarin.iOS), and Android (via Xamarin.Android) platforms.
```

Comment:
Windows Phone is EoS, so maybe we should remove that statement: https://support.microsoft.com/en-us/help/4036480/windows-phone-8-1-end-of-support-faq

Reply:
Done